src/bin/zmrabbitmqctl

```diff
@@ -0,0 +1,174 @@
+#!/bin/sh
+# 
+# ***** BEGIN LICENSE BLOCK *****
+# Zimbra Collaboration Suite Server
+# Copyright (C) 2005, 2006, 2007, 2008, 2009, 2010, 2011, 2012, 2013, 2014, 2016, 2021 Synacor, Inc.
+#
+# This program is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software Foundation,
+# version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License along with this program.
+# If not, see <https://www.gnu.org/licenses/>.
+# ***** END LICENSE BLOCK *****
+# 
+NAME=rabbitmq-server
+DAEMON=/opt/zimbra/common/sbin/${NAME}
+CONTROL=/opt/zimbra/common/sbin/rabbitmqctl
+DESC=rabbitmq
+USER=zimbra
+PID_FILE=/opt/zimbra/data/rabbitmq/pid
+LOCK_FILE=/opt/zimbra/data/rabbitmq/$NAME
+RABBITMQ_SERVER_CONSOLE_OUTPUT_DIR=/opt/zimbra/log/rabbitmq
+RABBITMQ_LOG_BASE=/opt/zimbra/log/rabbitmq
+RABBITMQ_STARTUP_TIMEOUT=600
+CONF_ENV_FILE=/opt/zimbra/common/lib/rabbitmq/rabbitmq-env.conf
```

Comment:
aren't we storing conf files in /opt/zimbra/conf?

Reply:
yes need store in /opt/zimbra/conf, updated changes for the same

Reply:
@silentsakky need one suggestion, currently we are getting /opt/zimbra/conf/rabbitmq-env.conf file from rabbitmq pacakge. In future if any changes introduced in rabbitmq-env.conf again we need to compile rabbitmq . I think this is not good approach.  In which repo we need to put rabbitmq-env.conf ?

Reply:
I don't think we have any repo where we can put conf files like that, so for now, let's go with current approach in future if need arises we will think about solving this problem